src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValue.cs

```diff
@@ -43,5 +43,7 @@ internal FormulaValue(IRContext irContext)
         public abstract object ToObject();
 
         public abstract void Visit(IValueVisitor visitor);
+
+        public abstract string ToExpression();
```

Comment:
> public abstract string ToExpression(); [](http://example.com/codeflow?start=7&length=39)

We should decide whether we just make this ToString() or create a new method.

Reply:
Also, we probably need the virtual tp accept a TextWriter or StringBuilder. 
And then have a ToString() helper for conveneince that creates the StringBuilder for us and calls that and returns the string,